src/test/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractFeaturesEngineTest.java

```diff
@@ -0,0 +1,66 @@
+package org.broadinstitute.hellbender.tools.variantdb.nextgen;
+
+import org.broadinstitute.hellbender.GATKBaseTest;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class ExtractFeaturesEngineTest extends GATKBaseTest {
+
+    @Test
+    public void testExtractFeaturesEngineQueryLabelCorrect() {
+        List<String> labelStringList = new ArrayList<String>();
+        labelStringList.add("labelkey=labelvalue");
+        Map<String, String> labelMap = ExtractFeaturesEngine.createQueryLabels(labelStringList);
+        Assert.assertEquals(labelMap.get("gvs_tool_name"), "extract-features");
+        Assert.assertEquals(labelMap.get("gvs_query_name"), "extract-features");
```

Comment:
don't you want to assert that your new label is also in there?

Reply:
hahaha true